docs/DETAILS.md

```diff
@@ -72,10 +72,18 @@ abort:
 Set the value of the `configMapField` to `"true"` to abort the validation. In all other cases (e.g. the `ConfigMap` is not found) the validation will execute normally.
 
 ## Logging
-Kubernetes Resource Validator uses the [`logr`](https://github.com/go-logr/logr) library as an interface to logging violations as well as other errors.
+Kubernetes Resource Validator uses the [`logr`](https://github.com/go-logr/logr) library as its logging interface.
 
 The actual logging implementation is set by users of this library. See the `logr` documentation for a partial [list of supported logging implementations](https://github.com/go-logr/logr#implementations-non-exhaustive).
 
+Note that the `logr` instance should be placed on the `ctx` parameter of the `validation.NewValidation()` function. For example:
```

Comment:
```suggestion
# Note that the `logr` instance should be placed on the `ctx` parameter of the `validation.NewValidation()` function. For example:
```

Reply:
# I think that formatting the entire line as heading level 1 is a bit much

Instead, I made the **note** bold.